src/test/ui/traits/object/enforce-supertrait-projection.stderr

```diff
@@ -0,0 +1,26 @@
+error[E0271]: type mismatch resolving `<dyn Trait<B = B, A = A> as SuperTrait>::A == B`
```

Comment:
Error message and spans kinda suck on this :/

Reply:
is it worth making some of the letters different or is the similarities part of the problem?

Reply:
Nah, the letters are unrelated. I can fix that. Just copied example code from the linked issue.